test/e2e/nightly/runtest.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
```

Comment:
Not sure this script is needed. It seems that @jkryl plans to run them [here](https://github.com/openebs/Mayastor/pull/597/files#diff-dfa8a5c1766e059a8823270f22b8bf652b3a34894d9d940e1ac8bbe7ca2a806eR86) and should be added [here](https://github.com/openebs/Mayastor/pull/597/files#diff-dfa8a5c1766e059a8823270f22b8bf652b3a34894d9d940e1ac8bbe7ca2a806eR8). If you think I'm right, please remove the script to avoid confusing people on which is used.

Reply:
Indeed. IMHO having one script for running tests and an option to run nightly tests is preferable solution.

Reply:
Removed